frontend/src/utils/emails/template-parser.ts

```diff
@@ -0,0 +1,70 @@
+// A naive template parser for the .template email files
+import { readFileSync } from 'fs'
+
+type EmailTemplate = {
+  subject: string
+  body: string
+}
+
+function readFile(filename: string) {
+  return readFileSync(`./bodies/${filename}`, 'utf-8')
```

Comment:
I don't believe `fs` is available for use on the client side. Though the type definitions exist, when actually trying to run the app this is not a function.